cmd/src/batch_common.go

```diff
@@ -477,7 +479,13 @@ func parseBatchSpec(file string, svc *service.Service) (*batcheslib.BatchSpec, s
 		return nil, "", errors.Wrap(err, "reading batch spec")
 	}
 
-	spec, err := svc.ParseBatchSpec(data)
+	p, err := filepath.Abs(file)
+	if err != nil {
+		return nil, "", errors.Wrap(err, "batch spec path")
+	}
+	dir, _ := filepath.Split(p)
```

Comment:
For clarity, I probably would have gone with:

```suggestion
	dir := filepath.Dir(p)
```

Unless there's a behavioural quirk in one of `Dir` or `Split` that I've forgotten. (Which is entirely possible!)